src/js/nav-secondary.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+/*global document window*/
+
+(function() {
+  const navigation = document.querySelector('[data-nav-secondary');
```

Comment:
```suggestion
  const navigation = document.querySelector('[data-nav-secondary]');
```

Minor: Missing closing `]` data attr selector

Reply:
Fixed.